helm/korifi/values.schema.json

```
@@ -579,6 +586,21 @@
         }
       },
       "type": "object"
+    },
+    "experimental": {
+      "properties": {
+        "managedServices": {
+          "properties": {
+            "include": {
+              "description": "Enable managed services support",
+              "type": "boolean"
+            }
+          },
+          "type": "object"
+        }
+      },
+      "description": "Experimental features. Make sure you do not enable those on production. No guarantee provided! Backwards incompatible changes in future are quite probable!",
```

```suggestion
      "description": "Experimental features. No guarantees are provided and breaking/backwards incompatible changes should be expected. These features are not recommended for use in production environments.",
```

Suggesting a minor wording tweak. Otherwise it all LGTM. 👍

---

Ugh, I have merged that without noticing your comment. I am goind to put that in the next PR